src/debian/13/helix/Dockerfile

```diff
@@ -0,0 +1,75 @@
+FROM library/debian:trixie AS base
+
+RUN apt-get update
```

Comment:
This adds bloat to the image.  It defeats the purpose of running `rm -rf /var/lib/apt/lists/*` later on.   Having a smaller image is more important that a faster image build.